pkg/base1/test-dbus-common.js

```diff
@@ -302,6 +304,58 @@ export function common_dbus_tests(channel_options, bus_name) { // eslint-disable
         received = false;
         await dbus.call("/otree/frobber", "com.redhat.Cockpit.DBusTests.Frobber", "RequestSignalEmission", [0]);
         assert.equal(received, false, "signal not received");
+        assert.equal(signals, 1, "received exactly one signal");
+
+        subscription = dbus.subscribe({
```

Comment:
What are you testing here by subscribing a second time?  I think this test will pass on both the C bridge and the Python bridge, no?

Reply:
It does fail on the Python bridge as the `add-match` is never removed after removing an extra subscription below.